examples/apps/ai-collab/package.json

```diff
@@ -29,7 +29,30 @@
 		"start": "next dev",
 		"start:server": "tinylicious"
 	},
-	"dependencies": {},
+	"c8": {
+		"all": true,
+		"cache-dir": "nyc/.cache",
+		"exclude": [
+			"src/test/**/*.*ts",
+			"dist/test/**/*.*js"
+		],
+		"exclude-after-remap": false,
+		"include": [
+			"src/**/*.*ts",
+			"dist/**/*.*js"
+		],
+		"report-dir": "nyc/report",
+		"reporter": [
+			"cobertura",
+			"html",
+			"text"
+		],
+		"temp-directory": "nyc/.nyc_output"
+	},
+	"dependencies": {
+		"@fluid-experimental/presence": "workspace:~",
+		"dotenv": "^16.4.5"
+	},
```

Comment:
Let's put these in devDependencies with everything else. It's a bit of a tricky decision if we should or shouldn't, but for this particular example app I'm trying to use it and see if we run into any issues.

Also, I don't think we need `dotenv`?

Reply:
Oh. I use it on set my environment variable locally. Remove it.